docs/components/buttons/README.md

```diff
@@ -576,8 +506,8 @@ In the below example, a "Download" button is rendered as a button with icon and
 </template>
 
 <script>
-import { CdrButton } from '@rei/cdr-button';
-import { CdrIcon } from '@rei/cdr-icon';
+import { CdrButton } from '@rei/cedar';
+import { CdrIcon } from '@rei/cedar';
```

Comment:
Should we combine these to be `import { CdrButton, CdrIcon } from '@rei/cedar'`?